README.md

```diff
@@ -47,26 +47,26 @@ IBM Cloud services:
 
 Service Name | Import Path
 --- | --- 
-[Case Management](https://cloud.ibm.com/apidocs/case-management) | ibm-platform-services/case-management/v1
-[Catalog Management](https://cloud.ibm.com/apidocs/resource-catalog/private-catalog) | ibm-platform-services/catalog-management/v1
-[Configuration Governance](https://cloud.ibm.com/apidocs/security-compliance/config) | ibm-platform-services/configuration-governance/v1
-[Enterprise Billing Units](https://cloud.ibm.com/apidocs/enterprise-apis/billing-unit) | ibm-platform-services/enterprise-billing-units/v1
-[Enterprise Management](https://cloud.ibm.com/apidocs/enterprise-apis/enterprise) | ibm-platform-services/enterprise-management/v1
-[Enterprise Usage Reports](https://cloud.ibm.com/apidocs/enterprise-apis/resource-usage-reports) | ibm-platform-services/enterprise-usage-reports/v1
-[Global Catalog](https://cloud.ibm.com/apidocs/resource-catalog/global-catalog) | ibm-platform-services/global-catalog/v1
-[Global Search](https://cloud.ibm.com/apidocs/search) | ibm-platform-services/global-search/v2
-[Global Tagging](https://cloud.ibm.com/apidocs/tagging) | ibm-platform-services/global-tagging/v1
-[IAM Access Groups](https://cloud.ibm.com/apidocs/iam-access-groups) | ibm-platform-services/iam-access-groups/v2
-[IAM Identity Service](https://cloud.ibm.com/apidocs/iam-identity-token-api) | ibm-platform-services/iam-identity/v1
-[IAM Policy Management](https://cloud.ibm.com/apidocs/iam-policy-management) | ibm-platform-services/iam-policy-management/v1
-[IBM Cloud Shell](https://cloud.ibm.com/apidocs/cloudshell) | ibm-platform-services/ibm-cloud-shell/v1
-[Open Service Broker](https://cloud.ibm.com/apidocs/resource-controller/ibm-cloud-osb-api) | ibm-platform-services/open-service-broker/v1
-[Posture Management](https://cloud.ibm.com/apidocs/security-compliance/posture) | ibm-platform-services/posture-management/v1
-[Resource Controller](https://cloud.ibm.com/apidocs/resource-controller/resource-controller) | ibm-platform-services/resource-controller/v2
-[Resource Manager](https://cloud.ibm.com/apidocs/resource-controller/resource-manager) | ibm-platform-services/resource-manager/v2
-[Usage Metering](https://cloud.ibm.com/apidocs/usage-metering) | ibm-platform-services/usage-metering/v4
-[Usage Reports](https://cloud.ibm.com/apidocs/metering-reporting) | ibm-platform-services/usage-reports/v4
-[User Management](https://cloud.ibm.com/apidocs/user-management) | ibm-platform-services/user-management/v1
+[Case Management](https://cloud.ibm.com/apidocs/case-management) | platform-services/case-management/v1
```

Comment:
@dpopp07 can confirm this, but I'm pretty sure that the `@ibm-cloud/` part is needed in an import path.